public/manifest.json

```diff
@@ -1,12 +1,14 @@
 {
-  "name": "GALOY MERCHANT APP",
-  "short_name": "POS",
-  "start_url": "/merchant/",
+  "name": "POS Cash Register",
+  "short_name": "Cash Register",
+  "description": "A Bitcoin POS cash register application that connects with the merchant's wallet",
+  "start_url": "/merchant/[username]",
+  "scope": "/",
   "display": "standalone",
   "background_color": "#536FF2",
   "theme_color": "#536FF2",
   "orientation": "portrait-primary",
-  "id": "/merchant/",
+  "id": "/merchant/[username]",
```

Comment:
I have been thinking whether we should keep both current design and new. I think we may want to have the new interface as the default and scrap the old one.

the reason is I think the new design is stricly better than the previous one, and keeping both leads to more code overhead to maintain over time. curious to take the opinion from others on this one

Reply:
@nicolasburtey The current tipping page includes code for exposing the users lnurl address in QR code format (good for merchants who want to support this).  I don't think the current POS implementation supports this yet?

Reply:
> The current tipping page includes code for exposing the users lnurl address in QR code format (good for merchants who want to support this)

this?

https://pay.bbw.sv/nicolas/print

This is a good point.

This would be trivial to add to the new POS.  Actually I think this function to be more relevant to be included in the new POS versus the former tipping page.